src/psyclone/parse/kernel.py

```diff
@@ -415,22 +416,25 @@ def get_stencil(metadata, valid_types):
 
 
 class Descriptor(object):
-    '''A description of how a kernel argument is accessed
+    ''' A description of how a kernel argument is accessed.
```

Comment:
"...accessed, constructed from the kernel metadata."

Reply:
Thanks for the suggestion, done.